### ISSUE TITLE
Something about bullets? Formatting?

### ISSUE THREAD

**Issue by nevanscott** [midnightbreakfast/web#75]
@ftpavlik I remember you saying something about maybe list formatting in one of the pieces, but I honestly can't remember right now. Could you update this issue on github when you get a chance to reflect the problem, and reassign this issue to me?

Thanks!

**Comment by ftpavlik** [comment]
It's in "Rooms We Die In," the list following "Imagine a room like this" should be indented and bulleted with em-dashes.

**Comment by nevanscott** [comment]
Check out:

http://preview.midnightbreakfast.com/rooms-we-die-in/

Does that work for you? I'd rather stick to the site style for lists, than do a dashed list. I feel like this achieves the same thing...

**Comment by ftpavlik** [comment]
That works just fine. For some reason the "I'm in a fucking point-and-click adventure game" in the paragraph above has lost its italics... Seems to be an isolated case.

**Comment by nevanscott** [comment]
Good catch, I'll fix that now. Just so you know, there are actually a few ways to achieve italics, and I went through and switched to the "proper" ones throughout. (That's why it changed.) The three options are `<em>` for emphasis; `<cite>` for citing a publication, movie, show, etc.; and `<i>` for something like "alternate voice or mood"—which I used for things like thoughts.

I just forgot to make it so that `<i>` elements are presented in italics... fixing now though.
